app/javascript/components/table/TableRow.vue

```diff
@@ -46,18 +58,39 @@
 <script>
 import mixinId from '../../mixins/mixin-ids'
 import ChartRowTarget from '../charts/chart-row-target/ChartRowTarget.vue'
+import Tooltip from '../tooltip/Tooltip'
 
 export default {
   name: 'TableRow',
 
-  components: { ChartRowTarget },
+  components: { ChartRowTarget, Tooltip },
 
   mixins: [ mixinId ],
 
   props: {
     row: {
       type: Object, // { title: '', url: '', stats: [{ charts: [{ title: '', value: '', target: '', colour: '' }] }] }
       required: true
+    },
+    tooltipArray: {
+      type: Array, // [ { id: String, title: String, text: String } ]
+      required: true
+    }
+  },
+
+  computed: {
+    totalStats () {
```

Comment:
personally statsCount or statsLength makes more sense to me, but could just be me. Change to one of these if you prefer, otherwise this is fine :)